avocado_vt/plugins/vt_init.py

```diff
@@ -0,0 +1,180 @@
+            settings.register_option(section, 'type_specific_only',
+                                     help_msg=help_msg, key_type=bool,
+                                     default=False)
+
+            help_msg = ('RAM dedicated to the main VM. Usually defaults to '
+                        '1024, as set in "base.cfg", but can be a different '
+                        'value depending on the various other configuration '
+                        'files such as configuration files under "guest-os" '
+                        'and test provider specific files')
+            settings.register_option(section, 'mem',
+                                     help_msg=help_msg,
+                                     default='')
+
+            help_msg = 'Architecture under test'
+            settings.register_option(section, 'arch',
+                                     help_msg=help_msg,
+                                     default=None)
+
+            help_msg = 'Machine type under test'
+            settings.register_option(section, 'machine_type',
+                                     help_msg=help_msg,
+                                     default=DEFAULT_MACHINE_TYPE)
+
+            help_msg = 'Nettype (bridge, user, none)'
+            settings.register_option(section, 'nettype',
+                                     help_msg=help_msg,
+                                     default='')
+
+            help_msg = 'Bridge name to be used if you select bridge as a nettype'
+            settings.register_option(section, 'netdst',
+                                     help_msg=help_msg,
+                                     default='virbr0')
+
+            # [vt.qemu] section
+            section = 'vt.qemu'
+
+            help_msg = 'Path to a custom qemu binary to be tested'
+            settings.register_option(section, 'qemu_bin',
+                                     help_msg=help_msg,
+                                     default=None)
+
+            help_msg = ('Path to a custom qemu binary to be tested for the '
+                        'destination of a migration, overrides qemu_bin for '
+                        'that particular purpose')
+            settings.register_option(section, 'qemu_dst_bin',
+                                     help_msg=help_msg,
+                                     default=None)
+
+            help_msg = 'Accelerator used to run qemu (kvm or tcg)'
+            settings.register_option(section, 'accel',
+                                     help_msg=help_msg,
+                                     default='kvm')
+
+            help_msg = ('Whether to enable vhost for qemu (on/off/force). '
+                        'Depends on nettype=bridge')
+            settings.register_option(section, 'vhost',
+                                     help_msg=help_msg,
+                                     default='off')
+
+            help_msg = 'Monitor type (human or qmp)'
+            settings.register_option(section, '',
```

Comment:
And here it looks that missing the option name `monitor`.

Reply:
Absolutely! Thanks!